docs/workflows.md

```diff
@@ -231,14 +274,16 @@ If you use the "static data" field (pencil), you have to write it out yourself.
 
 PS: it still only works with PREVIOUS nodes (nodes before the one you're editing)
 
-### Parsing JSON
+## Parsing JSON
 Parsing JSON is essential to be able to get API data. Shuffle uses it's own schema, which seems to work quite well. 
 
-$ 					= identifier for a previous node / execution argument 
-$exec 			= execution argument chosen
-$exec.name  = you choose the argument "name" from the execution argument
+```
+$node_1		= refers to a previously defined node called "node_1"
+$exec		= execution argument chosen
+$exec.name	= you choose the argument "name" from the execution argument
+```
 
-The first text after $ will always be the name of the previous node. If you add "." ($exec.name), it will look for "name" as a json value under "exec". If the node doesn't exist, it will just write $exec.name straight up.
+The first text after `$` will always be the name of the previous node. If you add "." ($exec.name), it will look for `"name"` as a json value under `"exec"`. If the node doesn't exist, it will just write `$exec.name` straight up.
```

Comment:
I'm assuming the markdown for code formatting will render correctly on the docs page.

Reply:
This is indeed a good idea to highlight them as code